tigerpath/majors_and_certificates/majors/COM_2018.json

```
@@ -0,0 +1,263 @@
+                "VIS 4*",
+                "VIS 5*"
+              ]
+            },
+            {
+              "name": "Regional or Ethnic Studies",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Comparative work in literature and regional or ethnic studies: two 300-, 400-, or 500-level courses in AFS, AAS, EAP, EAS, ECS, EPS, HLS, JDS, LAS, LAO, NES, or SAS. These two courses normally emerge from a single program, and have the same prefix (AFS, for instance). Alternatively, students on this track might take two 300-, 400-, or 500-level courses in either ENV, GSS, or URB; as elsewhere, courses should emerge from a single program, and have the same prefix. Those interested in these options should consult with COM’s Director of Undergraduate Studies to plan a course of study.",
+              "course_list": [
+                "AFS 3*",
+                "AFS 4*",
+                "AFS 5*",
+                "AAS 3*",
+                "AAS 4*",
+                "AAS 5*",
+                "EAP 3*",
+                "EAP 4*",
+                "EAP 5*",
+                "EAS 3*",
+                "EAS 4*",
+                "EAS 5*",
+                "ENV 3*",
+                "ENV 4*",
+                "ENV 5*",
+                "EPS 3*",
+                "EPS 4*",
+                "EPS 5*",
+                "GSS 3*",
+                "GSS 4*",
+                "GSS 5*",
+                "JDS 3*",
+                "JDS 4*",
+                "JDS 5*",
+                "LAS 3*",
+                "LAS 4*",
+                "LAS 5*",
+                "NES 3*",
+                "NES 4*",
+                "NES 5*",
+                "SAS 3*",
+                "SAS 4*",
+                "SAS 5*",
+                "URB 3*",
+                "URB 4*",
+                "URB 5*"
+              ]
+            },
+            {
+              "name": "Creative Arts",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Comparative work in literary study and the creative arts: two 300-, 400-, or 500-level courses in creative writing, screenwriting, translation, dance, theatrical or musical performance, visual arts, film, or video. These two courses normally emerge from a single program, and have the same prefix (DAN, for instance). Students entering COM must select the creative option provisionally, as final admission depends upon the formal acceptance of the creative thesis proposal by the Creative Writing program. Applications for the creative thesis are due in the spring of the junior year; interested students should be look over additional information on writing the creative senior thesis.",
+              "course_list": [
+                "CWR 3*",
+                "CWR 4*",
+                "CWR 5*",
+                "DAN 3*",
+                "DAN 4*",
+                "DAN 5*",
+                "MTD 3*",
+                "MTD 4*",
+                "MTD 5*",

```

What about MUS, which seems to be implied by "[musical performance](https://complit.princeton.edu/undergraduate-program/current-majors/requirements)"?
```suggestion
                "MTD 5*",
                "MUS 3*",
                "MUS 4*",
                "MUS 5*",
```